src/redux-js/index.js

```diff
@@ -1,6 +1,6 @@
-import ACTIONS from './actions/action';
-import { store } from './stores';
-
-window.store = store;
-window.requestLoadingAction = ACTIONS.requestLoadingAction;
-window.socialLoginServiceAction = ACTIONS.socialLoginServiceAction;
+// import ACTIONS from './actions/action';
```

Comment:
remove all commented code. it ain't needed if it is commented out

Reply:
I have acted accordingly.